spec/rocket_chat/messages/im_spec.rb

```diff
@@ -355,7 +340,7 @@
           status: 400
         )
 
-      stub_authed_request(:get, '/api/v1/im.counters?roomId=&username=')
+      stub_authed_request(:get, '/api/v1/im.counters?roomId=')
```

Comment:
Looks like the error message *should* be 
```
error: 'Query param "roomId" is required [error-room-param-not-provided]'
```
See https://github.com/RocketChat/Rocket.Chat/blob/d1384ef5cdfce2896b464771ab8f8b03d49a214e/apps/meteor/app/api/server/v1/im.ts#L144-L146

Reply:
fixed